modules/docker/Makefile.build

```diff
@@ -7,6 +7,7 @@ DOCKER_BUILD_FLAGS ?= --no-cache
 docker\:build: $(DOCKER)
 	$(call assert-set,DOCKER)
 	$(call assert-set,DOCKER_IMAGE_NAME)
+	git ls-files -s | awk '{print $1" "$4}' | sed s'/^..//' | xargs -n 2 echo chmod  | bash -x
```

Comment:
Sample output from `git ls-files -s | awk '{print $1" "$4}'`

```
100644 d14b848abdbf80350484d6161685cef94e65abb2 0       rootfs/usr/local/include/toolbox/helm/chart/kubernetes-dashboard
120000 8398ea3d372e12f10fae04c5af0d7a1689c65dbb 0       rootfs/usr/local/include/toolbox/helm/chart/lb
100644 83f4ecbbc5bee019b17a13436d25fb147ba294e8 0       rootfs/usr/local/include/toolbox/helm/chart/list/Makefile
120000 af1f1be2ab0fdcbf5138461791f0d90736db957f 0       rootfs/usr/local/include/toolbox/helm/chart/log
120000 9f7870ea04a3e6eb719a4e29d74c77ac6286c76a 0       rootfs/usr/local/include/toolbox/helm/chart/metrics
120000 a3c79b13b4bfe0654b03093bd8664d46679af445 0       rootfs/usr/local/include/toolbox/helm/chart/monitoring
100644 a04732be24ee84ecb8b5bd54ac8b4354b246e806 0       rootfs/usr/local/include/toolbox/helm/chart/nginx-ingress
100644 688bd98c1c3ecbeb9ed7af9940e69e068e5c1843 0       rootfs/usr/local/include/toolbox/helm/chart/route53-kubernetes
120000 3f9045156fe1bf799001631c79aa29762861736b 0       rootfs/usr/local/include/toolbox/helm/chart/ssh
100644 cdf6ae69f28e3f35896ccfdd3cb8c3b8281a3dfe 0       rootfs/usr/local/include/toolbox/helm/chart/syslog-forwarder
120000 f6c7bd5fefe9c7b10e36eff9fb091e324f99187b 0       rootfs/usr/local/include/toolbox/helm/chart/tls
120000 49a6a0758abf3848a5539a4e73c30280974431bd 0       rootfs/usr/local/include/toolbox/helm/chart/ui
```

so we strip of the first to characters to get the mode. Then construct the `chmod` command and send that into bash.